src/modules/home/components/property/propertyCard.tsx

```diff
@@ -1,16 +1,115 @@
 import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
-import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
+import { Link } from 'react-router-dom';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import 'swiper/css';
+import 'swiper/css/effect-fade';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+import { EffectFade, Navigation, Pagination } from 'swiper/modules';
+
+type Accommodation = {
+  id: number;
+  name: string;
+  description: string;
+  images: string[];
+  location: string;
+  price_per_night: number;
+  type: string;
+  amenities: string[];
+  status: string;
+};
+
+export function ApartmentCard(apartment: Accommodation) {
+  const handleFavoriteClick = (event: React.MouseEvent) => {
+    event.stopPropagation();
+    event.preventDefault();
+    console.log('Favorite icon clicked');
+  };
 
-export function ApartmentCard() {
   return (
-    <Card>
-      <CardMedia component="img" height="140" image="https://placehold.co/600x400" alt="Apartment" />
-      <CardContent>
-        <Typography variant="h5">Apartment</Typography>
-        <Typography>Beautiful apartment in the city center</Typography>
-      </CardContent>
-    </Card>
+    <Link to={`/apartment/${apartment.id}`} style={{ textDecoration: 'none' }}>
+      <Card sx={{ boxShadow: 'none', position: 'relative' }}>
+        <FavoriteBorderIcon
+          onClick={handleFavoriteClick}
+          sx={{
+            position: 'absolute',
```

Comment:
Please take a look at the comment I left here and follow this flow:
https://github.com/vention-js-lab/ventioners-stayhub-frontend/pull/5#discussion_r1838030972